### DISCUSSION TITLE
Latest's Svelte release adds new Ally checks

### DISCUSSION THREAD

**Discussion by rhscjohn-dev** [saadeghi/daisyui#1339] [Q&A] [answered]
Just upgraded to Svelte version 3.44.0.  Looks like they have recently added some new Ally checks.

So far, the Daisy component - Dropdown,  is the most effected.

See [Svelte REPL](https://svelte.dev/repl/b44a404d94db47faabdafa1ee9a78481).

I will post here if I come across any more components.

**Comment by saadeghi** [comment, marked as answer]
Please read this: https://github.com/saadeghi/daisyui/issues/1285#issuecomment-1293588885
